{
  "parameter(s)": [
    -936
  ],
  "test_matrix_of_single_number_equal_to_sum": [
    "failed",
    "assert (array([0, 1]),) == [[[-936]]]\n  (pytest_assertion plugin: representation of details failed: /data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/assertion/util.py:346: ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all().\n   Probably an object has a faulty __repr__.)",
    0.00076
  ],
  "test_matrix_of_single_number_and_zeros": [
    "failed",
    "assert 1 == (2 * 2)\n +  where 1 = len((array([0, 2]),))\n +    where (array([0, 2]),) = submatrix_with_particular_sum(matrix([[-936,    0],\\n        [   0,    0]]))",
    0.00086
  ],
  "test_matrix_of_several_numbers": [
    "failed",
    "assert not (array([3]),)\n +  where (array([3]),) = submatrix_with_particular_sum(matrix([[1, 2, 3, 4, 5, 6, 7, 8, 9],\\n        [1, 2, 3, 4, 5, 6, 7, 8, 9],\\n        [1, 2, 3, 4, 5, 6, 7, 8, 9]]))",
    0.00101
  ]
}